ports/atmel-samd/boards/metro_m4_express/mpconfigboard.h

```diff
@@ -5,16 +5,16 @@
 
 // This is for Rev F which is green
 
-#define MICROPY_HW_LED_TX   PIN_PA27
-#define MICROPY_HW_LED_RX   PIN_PB06
+#define MICROPY_HW_LED_TX   (&pin_PA27)
```

Comment:
These are commented out on M0, but not here. Why the difference?

Reply:
I think its a mistake. I do it to make sure SWD works on PA31. Re-enabled.